shared/constants/network.ts

```diff
@@ -1146,10 +1146,12 @@ export const CHAIN_ID_TOKEN_IMAGE_MAP = {
 /**
  * A mapping for networks with enabled profolio landing page to their URLs.
  */
-export const CHAIN_ID_PROFOLIO_LANDING_PAGE_URL_MAP: Record<Hex, string> = {
+export const CHAIN_ID_PROFOLIO_LANDING_PAGE_URL_MAP: Record<string, string> = {
```

Comment:
typo:
```suggestion
export const CHAIN_ID_PORTFOLIO_LANDING_PAGE_URL_MAP: Record<string, string> = {
```

Same typo is also in line 1147.